README.md

```diff
@@ -86,7 +86,7 @@ version: 2
 
 models:
   - name: model_name
-    tests:
+    data_tests:
```

Comment:
Let's leave the README as-is right now so that the examples work across the broadest range of dbt-core.

But we can definitely revisit making this update in a separate PR, especially when there aren't any supported versions of dbt-core that allow `tests:` within `dbt_project.yml`.